README.md

```diff
@@ -46,7 +46,7 @@ We aim the tests to be fair. Our library puts performance as the #1 priority, wh
 * Allows you to apply patches on object trees for incoming traffic.
 * Allows you to freely manipulate object trees and then generate patches for outgoing traffic.
 * Tested in IE 8-11, Firefox, Chrome, Safari and Node.js
-
+* **NEW** It supports ES6 proxies. (Not in IE and Oprah Mini). 
```

Comment:
hahahahahahahahahhaha the Oprah  thing cracked me up. Thanks for the fast review!